dss-api

```diff
@@ -1,9 +1,31 @@
 #!/usr/bin/env python
 
+"""
+Entry point for starting a local test HCA DSS API server.
+"""
+
+import os
+import sys
 import logging
+import argparse
+
+from chalice.cli import CLIFactory, run_local_server
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument("--port", type=int, default=5000)
+parser.add_argument("--no-debug", dest="debug", action="store_false")
+parser.add_argument("--project-dir", help=argparse.SUPPRESS,
+                    default=os.path.join(os.path.dirname(__file__), "chalice"))
+parser.add_argument("--log-level",
```

Comment:
By logging contexts, you mean logger names, right? The standard idiom is to get the logger to log with on a per-package or per-module basis using `logger = logging.getLogger(__name__)`. This setting affects the default log verbosity for all of them. It's probably not worth exposing ability to set log levels for each of them individually, since that gets kinda messy quickly.